configure.ac

```diff
@@ -712,6 +718,61 @@ esac
 AC_SUBST(D2COMPILER)
 AC_SUBST(DLIBPREFIX)
 
+#---------------------------------------------------------------
+# Look for Fortran
+#---------------------------------------------------------------
+
+AC_ARG_WITH(fortran,
+AS_HELP_STRING([--without-fortran], [Disable Fortran])
+AS_HELP_STRING([--with-fortran=path], [Set location of Fortran compiler]),
+[FORTRAN="$withval"], [FORTRAN="$alllang_default"])
+
+if test x"${FORTRAN}" = xno; then
+  AC_MSG_NOTICE([Disabling Fortran])
+  FC=
+else
+  if test -n "$FCFLAGS" && test "x$GCC" = xyes; then
+    FCFLAGS="-std=f2003 -Wall -Wimplicit-procedure -Wimplicit-interface"
+  fi
+
+  if test "x$FORTRAN" = xyes; then
+    AC_PROG_FC dnl This command also sets FCFLAGS
+  else
+    FC=$FORTRAN
+    FCFLAGS=$FCFLAGS
+  fi
+
+  if test -n "$FC"; then
+    AC_FC_LIBRARY_LDFLAGS dnl Set up Fortran library flags
+  fi
+
+  # FORTRANLDSHARED is the ld *command* used to create shared library
+  # This must make a dynamic library as opposerd to a bundle.
+  case $host in
+    *-*-darwin*)
```

Comment:
Just a minor tip to make things a little more readable - you can use `$host_os` to get just the last part of the `$host` triplet, and avoid needing the `*-*-` on your patterns (here and below).

Reply:
Thanks for the heads up. This `$host` and the `*-*-OS` idiom is used all over the config.ac file but `$host_os` isn't used anywhere. Would it be better to keep the current use for consistency and add a later simplification to all the other target languages at the same time? (Or have a separate PR on master for making that change?) I don't want to accidentally introduce an incompatibility with some version of autoconf.

Reply:
Probably makes sense to keep consistency and address this separately.

There's really no portability worry about `$host_os` though - it's been available in autoconf since 1994 and SWIG requires at least autoconf 2.60 (released 2006).